scripts/provision.sh

```diff
@@ -220,7 +248,25 @@ upload_ssl_certificate() {
   fi
 }
 
+start_init() {
+  echo
+  echo "Starting Habitat Supervisor"
+  create_users
+  sudo systemctl start hab-sup
+  sleep 2
+}
+
+start_frontend() {
+  echo
+  echo "Starting Builder Frontend Services"
+  start_memcached
+  start_api
+  start_api_proxy
+}
+
 start_builder() {
+  echo
+  echo "Starting Builder Services"
   if [ "${RDS_ENABLED:-false}" = "false" ]; then
```

Comment:
It's not part of this pr, but should these also be `==`?

Reply:
Nope. As I understand it, the `[]` in bash are a synonym for `test` and for the `test` syntax this is correct.